internal/remoteconfig/remoteconfig.go

```diff
@@ -233,6 +234,7 @@ func (c *Client) updateState() {
 	}
 
 	if body := string(respBody); body == `{}` || body == `null` {
+		log.Error("remoteconfig: empty body %s\n", body)
```

Comment:
Let's change the other two logs here to `Debug` and remove the error log for empty body.